src/main/java/com/google/codeu/data/Event.java

```diff
@@ -0,0 +1,53 @@
+/** Event class */
+
+package com.google.codeu.data;
+
+public class Event {
+  static int totalEvents = 0;
```

Comment:
Hmm, I'm actual not sure if keeping track of the number of events as a static field will work since it depends on how the appengine server works.  If our website in exactly one process on one server, then it would be ok, but if the website is replicated on multiple servers, then there would be multiple copies of this static field and they wouldn't stay in sync.  Since, in general, it's not possible to rely on the static field, let's follow the pattern used the [other data classes](https://github.com/mrios730/Team-25/blob/master/src/main/java/com/google/codeu/data/Message.java#L24) and initialize the `id` to be `UUID.randomUUID()` and remove `totalEvents`.

Reply:
That makes a lot more sense! I was worried about how to make each event have a distinct `id`. Thanks! I'll update the file.